### PR TITLE
common/prometheus-server: forward multiple scrape alerts

### PR DESCRIPTION
**Pull request by richardtief**

### CODE REVIEW

File: common/prometheus-server/templates/alerts/_prometheus.alerts.tpl [hunk 1]
```diff
@@ -11,6 +11,9 @@ Note: The pods define the 'alert-service' label but Prometheus replaces the hyph
 {{- define "alertServiceLabelOrDefault" -}}
 "{{`{{ if $labels.label_alert_service }}`}}{{`{{ $labels.label_alert_service}}`}}{{`{{ else }}`}}{{ required "default value is missing" . }}{{`{{ end }}`}}"
 {{- end -}}
+{{- define "alertSupportGroupOrDefault" -}}
+"{{`{{ if $.Values.alerts.support_group }}`}}{{`{{ $.Values.alerts.support_group }}`}}{{`{{ else if $labels.ccloud_support_group }}`}}{{`{{ $labels.ccloud_support_group }}`}}{{`{{ else }}`}}{{ required "default value is missing" . }}{{`{{ end }}`}}"
+{{- end -}}
```
Comment (IvoGoman) [inline common/prometheus-server/templates/alerts/_prometheus.alerts.tpl:16]: I am not sure about the order of `$.Values.alerts.support_group` and `$labels.ccloud_support_group`.
Shouldn't the more specific support group of the metric take precedence over the support group from the values?

**Review by IvoGoman — CHANGES_REQUESTED** [review]

File: common/prometheus-server/templates/alerts/_prometheus.alerts.tpl (hunk 1 above)
Comment (richardtief) [reply common/prometheus-server/templates/alerts/_prometheus.alerts.tpl]: Yes, you are right. The label takes the lead.

Comment (richardtief) [reply common/prometheus-server/templates/alerts/_prometheus.alerts.tpl]: The values entry can then be omitted, which would only make sense in rare cases.

**Review by IvoGoman — APPROVED** [review]
Thank you 👍

**Pull request merged**